README.md

```diff
@@ -4,21 +4,37 @@ The implementation of the DAO dashboard.
 
 [![ContinuousBuild](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml)
 
-## Running dashboard on Mainnet in docker container
+## Running the dashboard on Mainnet in a Docker container
 
-The decentralized approach of being a DAO member is to run API3 dashboard on your local machine. For that you will just
-need `git` and `docker`. Note, that the production source code is on the `production` branch.
+The decentralized approach to being a DAO member is to run API3 dashboard on your local machine. For that you will just
+need `docker` and, optionally, `git`.
 
+There are two approaches to running the dashboard locally. The first involves using the prebuilt Docker image, while the
+second involves building the image from source. In either case, the end result is a API3 dashboard running on port 7770
+of your localhost where it is safe to connect your wallet. Once you are finished interacting with the dashboard, the
+container can be stopped using `docker stop api3-dao-dashboard`.
+
+### Running the prebuilt image
+
+The simplest, though less trustless, way to run the dashboard is using the prebuilt Docker image. This image is built
```

Comment:
`though less trustless` sounds too fearful. I am not sure about this, since we are OK with using Airnode via Docker and here we have similar risks (Dockerhub being malicious which realistically won't happen).

Reply:
I generally agree here. We could add `slightly` to lessen the frightfulness of the language or remove it.

Reply:
Removed